src/AutoRest.CSharp.V3/Output/Models/Requests/LongRunningOperation.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using AutoRest.CSharp.V3.Output.Models.Responses;
+using AutoRest.CSharp.V3.Output.Models.Shared;
+using Azure.Core;
+
+namespace AutoRest.CSharp.V3.Output.Models.Requests
+{
+    internal class LongRunningOperation
+    {
+        public LongRunningOperation(Method originalMethod, Response originalResponse, string name, Parameter[] createParameters, FinalStateVia finalStateVia)
+        {
+            OriginalMethod = originalMethod;
+            OriginalResponse = originalResponse;
+            Name = name;
+            CreateParameters = createParameters;
+            FinalStateVia = finalStateVia;
+        }
+
+        public string Name { get; }
+        public Method OriginalMethod { get; }
```

Comment:
Is this a method that initiates operation?

Reply:
It is the original operation, so yes. The nomenclature in the LRO documentation uses both 'original request' and 'initial request' to describe this. In my diagram, I used 'original operation' to differentiate between that and a 'polling operation'.

Reply:
Can we rename it to InitialRequest and FinalResponse?